src/ui/components/shared/PortalDropdown.tsx

```diff
@@ -50,7 +50,7 @@ export default function PortalDropdown(props: PortalDropdownProps) {
     <div className="portal-dropdown-wrapper">
       <button
         type="button"
-        className={`expand-dropdown flex w-full ${buttonStyle}`}
+        className={`expand-dropdown mr-1 flex w-full px-2 py-1 ${buttonStyle}`}
```

Comment:
Note that #8466 is reverting the `px-2` portion of this change as it seemed to cause a visual regression reported by FE-1098.

Reply:
Quote from me:
"And when a bug slips through, the responsibility lands on that [person] to fix it."

Also me:
"mr-1 flex w-full px-2 py-1"

That was my bad, thanks for the catch Brian.

Reply:
No worries. It's really easy to do. Hopefully we'll get more and more screenshot tests setup and things like this will be easier to spot sooner.